train.py

```diff
@@ -291,13 +291,12 @@ def _create_dataloaders():
         if sparsification_manager and sparsification_manager.starting_qat(epoch):
             sparsification_manager.disable_ema_amp(ema, amp, scaler)
             # Rescale batch size for QAT
-            if opt.batch_size == -1:
-                batch_size, accumulate = sparsification_manager.rescale_gradient_accumulation(
-                    batch_size=batch_size, 
-                    accumulate=accumulate, 
-                    image_size=imgsz
-                )
-                train_loader, dataset, val_loader = _create_dataloaders()
+            batch_size, accumulate = sparsification_manager.rescale_gradient_accumulation(
+                batch_size=batch_size, 
+                accumulate=accumulate, 
+                image_size=imgsz
+            )
+            train_loader, dataset, val_loader = _create_dataloaders()
```

Comment:
Is this supposed to be in this PR? is it safe to remove the opt.batch_size -1 check?

Reply:
Yep, this is the first use-case covered!

`Manual batch size will now also trigger batch size re-calculation on QAT start`